webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/appconfig/ConfigService.scala

```diff
@@ -441,6 +445,7 @@ class LDAPBasedConfigService(
        rudder.generation.delay=0s
        rudder.generation.trigger=all
        node.accept.duplicated.hostname=false
+       rudder.compute.dyngroups.max.parallelism=1
```

Comment:
I think that for consistancy sack, it should be like `rudder.generation.max.parallelism`, ie accepting either an integer like here, or `xF`, with `F` a float and `x` the character `x`, meaning `F time the number of CPU available`. The same parsing logic can be used (and can even be factored out in an utility object, which would also ensure to always return an integer at least equals to `1`).
And with that, setting it to `x0.5` (ie, `number CPU / 2`) seems to be a good default.

Reply:
it's not really generation, as it is to compute dynamic groups (so also done outside of policy generation)

i'll change the xF

Reply:
I'm not sure i'd like to change the default to something larger than 1, as it depends on the size of the LDAP connection pool which is 2 by default